draft-marchan-kdl2.md

```diff
@@ -39,9 +79,9 @@ the directions if the data stream were only ASCII text. They do not refer
 to the writing direction of text, which can flow in either direction,
 depending on the characters used.
 
-## Components
+# Components
 
-### Document
+## Document
 
 The toplevel concept of KDL is a Document. A Document is composed of zero or
 more [Nodes](#node), separated by newlines and whitespace, and eventually
```

Comment:
Two things:

1. Looks like this specific instance is missing the section link. Perhaps there are other cases?

2. On a more general note, I notice that there's some inconsistency as to events we do `[Foo](#foo) ([Section 1](#section1))` or just `Foo ([Section 1](...))`. I would prefer we normalize around one. The second (just one link) probably makes the most sense, since linking the name itself seems redundant if you're going to link the section anyway.